src/js/SignUpUI.js

```diff
@@ -0,0 +1,56 @@
+import AuthAPI from './AuthAPI';
+
+export default class SignUp {
+  user = document.querySelector('#username');
+
+  pass = document.querySelector('#signup-pass');
+
+  email = document.querySelector('#email');
+
+  userError = document.querySelector('#username-err');
+
+  emailError = document.querySelector('#email-err');
+
+  signupError = document.querySelector('#signup-err');
+
+
+
+  signUp() {
+    const { user, pass, email, userError, emailError, signupError, errorLog} = this;
+    const  regExpEmail = new RegExp(
+        /^([a-zA-Z0-9_\-.]+)@([a-zA-Z0-9_\-.]+)\.([a-zA-Z]{2,5})$/
+    );
+    let errors = 0;
+    if (user.value.length < 1) {
+      errorLog(`There is an empty field`, userError)
+      errors++;
+    }
+    if (pass.value.length < 1 ) {
+      errorLog(`There is an empty field`, signupError)
+      errors++;
+    }
+    if (!regExpEmail.test(email.value)) {
+      errorLog(`Invalid email example@test.io`, emailError)
+      errors++;
+    }
+    if (errors === 0){
+      AuthAPI.registration({
+        username: user.value,
+        email: email.value,
+        password: pass.value,
```

Comment:
```suggestion
        username: user.value.trim(),
        email: email.value.trim(),
        password: pass.value.trim(),
```
we should trim the sent data